Mathlib/Algebra/Algebra/Subalgebra/Basic.lean

```diff
@@ -741,7 +741,7 @@ noncomputable def ofInjectiveField {E F : Type*} [DivisionRing E] [Semiring F] [
 
 /-- Given an equivalence `e : A ≃ₐ[R] B` of `R`-algebras and a subalgebra `S` of `A`,
 `subalgebra_map` is the induced equivalence between `S` and `S.map e` -/
-@[simps!]
+@[simps!, nolint simpNF]
```

Comment:
A reminder from the other PR: this is a no-op, and should be reverted

Reply:
Just do it?
```suggestion
@[simps!]
```

Reply:
There are a handful of these throughout